libs/perun/dialogs/src/lib/delete-entity-dialog/delete-entity-dialog.component.html

```diff
@@ -1,24 +1,34 @@
 <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
 <div *ngIf="!force && !loading">
-  <h1 mat-dialog-title>{{'DIALOGS.DELETE_ENTITY.DELETE' | translate}} {{entityType}}</h1>
+  <h1 mat-dialog-title>
+    {{'DIALOGS.DELETE_ENTITY.DELETE' | translate: {type: this.anonymize | getStringForDialogType | titlecase} }}
+    {{entityType}}
+  </h1>
   <div class="dialog-container" mat-dialog-content>
     <div *ngIf="!disableForce" class="mb-4">
-      {{'DIALOGS.DELETE_ENTITY.BASIC' | translate}}
+      {{'DIALOGS.DELETE_ENTITY.BASIC' | translate: {type: this.anonymize | getStringForDialogType} }}
       {{'DIALOGS.DELETE_ENTITY.ONLY' | translate}}
       {{entityType}}
       <span class="font-weight-bold">{{'DIALOGS.DELETE_ENTITY.OR' | translate}}</span>
       {{entityType}}
       {{'DIALOGS.DELETE_ENTITY.RELATIONS' | translate}}?
     </div>
     <div *ngIf="disableForce" class="mb-4">
-      {{'DIALOGS.DELETE_ENTITY.BASIC' | translate}}
+      {{'DIALOGS.DELETE_ENTITY.BASIC' | translate: {type: this.anonymize | getStringForDialogType} }}
       {{entityType}}
       ?
     </div>
     <table [dataSource]="entityNames" class="w-100 table-margin" mat-table>
       <ng-container matColumnDef="name">
         <th *matHeaderCellDef mat-header-cell></th>
-        <td *matCellDef="let entity" mat-cell>{{entity.name}}</td>
+        <td *matCellDef="let entity" mat-cell>
+          <span *ngIf="anonymize">
+            {{entity | userFullName}}
```

Comment:
Something like `entity.name ?? (entity | userFullname)` could spare us the double span unless we excpet more branching will be needed.